packages/formation/sass/modules/_m-responsive-tables.scss

```diff
@@ -43,6 +37,7 @@ table.responsive {
     td {
       border: none;
       font-size: 16px;
+      display: flex;
```

Comment:
This lets the row cells stack vertically instead of being shoved together horizontally, like a typical table row.